src/Routes/Home.jsx

```diff
@@ -1,11 +1,22 @@
-import React from "react";
+import { React, useContext } from "react";
+import Header from "../Components/Header/Header.jsx";
+import Testimonials from "../Components/Testimonial-Cards/Testimonials.jsx";
+import Pricing from "../Components/Pricing";
+import AboutUs from "../Components/Body/AboutUs";
+import ResponsiveCarousel from "../Components/Carousel/Carousel.jsx";
+import { AppContext } from "../AppContext.jsx";
 
-function Home () {
-    return (
-        <div className="home">
-            <h1>Home</h1>
-        </div>
-    )
+function Home() {
+  const { isDark } = useContext(AppContext);
+  return (
+    <div className="">
+      <Header isDark={isDark} />
```

Comment:
**todo:** `Header` braucht diese Prop nicht. `mode--dark` wird bereits auf .app gesetzt und kann deshalb aus der ganzen app selektiert werden. Der Selektor in Header.css muss dann nur von `&.mode--dark {` auf `.mode--dark & {` geändert werden. Dann brauchst du in der Header.jsx nicht noch mal eine Klasse für den Dark mode setzen